docs/kratos/guides/deploy-kratos-example.mdx

```diff
@@ -470,245 +475,238 @@ nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
+       proxy_redirect          off;
+       proxy_set_header        Host            $host;
+       proxy_set_header        X-Real-IP       $remote_addr;
+       proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+     }
+   ...
+     error_page 401 = @error401;
+     # Catch if 401/unauthorized and redirect for login
+     location @error401 {
+       return 302 https://accounts.example.com/auth/login;
+     }
+
+   ```
+
+3. Test the Nginx configuration:
+
+   ```bash
+   nginx -t
+   ```
+
+4. You should get the following response if the configuration is correct:
+
+   ```bash
+   nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
+   nginx: configuration file /etc/nginx/nginx.conf test is successful
+   ```
+
+5. Reload the Nginx and Ory Kratos services:
+
+   ```bash
+   service nginx reload
+   service kratos restart
+   ```
 
 Open `https://accounts.example.com/auth/login` and you should see the Login Screen.
 
 ## Secure secrets
 
 We covered a basic deployment of Ory Kratos to production with Nginx. However, the configuration of Kratos itself is not
-production-ready. We need to:
+production-ready.
 
-1. Set up secure keys.
-1. Run Ory Kratos in production mode.
+We need to set up secure keys and run Ory Kratos in production mode:
 
-Generate secure keys using the following [`openssl`](https://www.openssl.org/docs/manmaster/man1/openssl.html) command:
+1. Generate secure keys using the following [`openssl`](https://www.openssl.org/docs/manmaster/man1/openssl.html) command:
 
-```bash
-openssl rand -base64 32
-```
-
-Edit `/opt/kratos/config/kratos.yml` file and replace the following values:
-
-```yaml
-secrets:
-  cookie:
-    - PLEASE-CHANGE-ME-I-AM-VERY-INSECURE
-  cipher:
-    - 32-LONG-SECRET-NOT-SECURE-AT-ALL
-```
+   ```bash
+   openssl rand -base64 32
+   ```
 
-with unique and generated strings:
+2. Edit `/opt/kratos/config/kratos.yml` file and replace the following values with unique and generated strings: :
 
-```yaml
-secrets:
-  cookie:
-    - xgrxYZLM0jGgJqijvvf73x30URJkRENoCm2ExQw9+CE=
-  cipher:
-    - rtxrVjiWwe85nxB9XeWoDP8U17+sT6kHo0E1t3Tr5uY=
-```
+   ```diff
+   secrets:
+     cookie:
+   -    - PLEASE-CHANGE-ME-I-AM-VERY-INSECURE
+   +    - xgrxYZLM0jGgJqijvvf73x30URJkRENoCm2ExQw9+CE=
+     cipher:
+   -    - 32-LONG-SECRET-NOT-SECURE-AT-ALL
+   +    - rtxrVjiWwe85nxB9XeWoDP8U17+sT6kHo0E1t3Tr5uY=
+   ```
 
-Change `log.leak_sensitive_values` to false, to not leak any sensitive values like secrets in the logs:
+   3. Change `log.leak_sensitive_values` to false, as to not leak any sensitive values like secrets in the logs:
 
-```diff
-log:
--  leak_sensitive_values: true
-+  leak_sensitive_values: false
-```
+   ```diff
+   log:
+   -  leak_sensitive_values: true
+   +  leak_sensitive_values: false
+   ```
 
-Save changes and restart Ory Kratos:
+3. Save changes and restart Ory Kratos:
 
-```
-systemctl restart kratos
-```
+   ```
+   systemctl restart kratos
+   ```
```

Comment:
Something weird going on here, pls have a look at the preview and do a fixaroo

<img width="702" alt="image" src="https://user-images.githubusercontent.com/29116492/180948962-9c91999e-03bc-4527-9feb-103193096979.png">